samples/matter/window_covering/sample.yaml

```diff
@@ -0,0 +1,35 @@
+sample:
+  name: Matter Window Covering
+  description: Matter Window Covering example
+tests:
+  matter.lock.debug:
```

Comment:
change for matter.window_covering.debug or matter/windcov.debug

Reply:
Done